explorer/ExplorerConstants.ts

```diff
@@ -8,6 +8,13 @@ export enum ExplorerControlType {
     Dropdown = "Dropdown",
 }
 
+export enum ChartCreationMode {
+    WithGrapherId = "WithGrapherId",
+    WithIndicatorIds = "WithIndicatorIds",
+    WithManuallyProvidedData = "WithManuallyProvidedData",
```

Comment:
I know that it's called that in the grapher property, but when I first read it I believed this would be about manually creating a table and passing it in (which we sometimes do for test case; and we also support inline tables in explorer configs).

Maybe makes sense to rename it to `FromExplorerTableColumnSlugs` or something like that, and maybe also rename the enum itself to `ExplorerChartCreationMode` to highlight that it doesn't apply to graphers at all.